apps/site/pages/en/blog/release/v23.3.0.md

```diff
@@ -0,0 +1,147 @@
+---
+date: '2024-11-20T23:01:40.107Z'
+category: release
+title: Node v23.3.0 (Current)
+layout: blog-post
+author: Rafael Gonzaga
+---
+
+## 2024-11-20, Version 23.3.0 (Current), @RafaelGSS
+
+### Notable Changes
+
+- \[[`5767b76c30`](https://github.com/nodejs/node/commit/5767b76c30)] - **doc**: enforce strict policy to semver-major releases (Rafael Gonzaga) [#55732](https://github.com/nodejs/node/pull/55732)
+- \[[`ccb69bb8d5`](https://github.com/nodejs/node/commit/ccb69bb8d5)] - **(SEMVER-MINOR)** **src**: add cli option to preserve env vars on dr (Rafael Gonzaga) [#55697](https://github.com/nodejs/node/pull/55697)
```

Comment:
```suggestion
- \[[`ccb69bb8d5`](https://github.com/nodejs/node/commit/ccb69bb8d5)] - **(SEMVER-MINOR)** **src**: add cli option to preserve env vars on diagnostic reports (Rafael Gonzaga) [#55697](https://github.com/nodejs/node/pull/55697)
```

Reply:
lol I did that change before commit and push. I have no idea why it didn't show up